framework/src/test/java/org/tron/core/db/AccountTraceStoreTest.java

```diff
@@ -0,0 +1,59 @@
+package org.tron.core.db;
+
+import com.google.common.primitives.Bytes;
+import com.google.common.primitives.Longs;
+import com.google.protobuf.ByteString;
+import javax.annotation.Resource;
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.tron.common.BaseTest;
+import org.tron.common.utils.ByteArray;
+import org.tron.core.Constant;
+import org.tron.core.capsule.AccountCapsule;
+import org.tron.core.config.args.Args;
+import org.tron.core.exception.BadItemException;
+import org.tron.core.exception.ItemNotFoundException;
+import org.tron.core.store.AccountIndexStore;
+import org.tron.core.store.AccountTraceStore;
+import org.tron.protos.Protocol.AccountType;
+
+public class AccountTraceStoreTest extends BaseTest {
+
+  private static String dbDirectory = "db_AccountTraceStore_test";
+  @Resource
```

Comment:
`dbDirectory` is not being used and should be deleted. There are a lot of the same issues behind it, please fix them all together.